pacman-rules/lib/rules.ex

```diff
@@ -1,33 +1,17 @@
 defmodule Rules do
   def eat_ghost?(power_pellet_active, touching_ghost) do
-     if power_pellet_active && touching_ghost do
-       true
-     else
-       false
-     end
+     power_pellet_active && touching_ghost 
   end
 
   def score?(touching_power_pellet, touching_dot) do
-     if touching_power_pellet || touching_dot do
-       true
-     else
-       false
-     end
+     touching_power_pellet || touching_dot 
   end
 
   def lose?(power_pellet_active, touching_ghost) do
-     if power_pellet_active== false && touching_ghost do
-        true
-     else
-        false
-     end
+     power_pellet_active== false && touching_ghost 
```

Comment:
Not recommended to write the condition like this. You can simply write:
```elixir
!power_pellet_active
```
BTW, I don't approve of the spacing in this line, I'd be less worried if you take care of your code style. This easily could've been:
```elixir
power_pellet_active == false && touching_ghost 
# instead of
power_pellet_active== false && touching_ghost 
```